pkg/tests/apis/folder/folders_test.go

```diff
@@ -386,6 +386,22 @@ func doFolderTests(t *testing.T, helper *apis.K8sTestHelper) *apis.K8sTestHelper
 		for _, uid := range uids {
 			getFromBothAPIs(t, helper, client, uid, nil)
 		}
+
+		// PUT :: Update the title
+		updated, err := client.Resource.Update(context.Background(),
+			helper.LoadYAMLOrJSONFile("testdata/folder-test-replace.yaml"),
+			metav1.UpdateOptions{},
+		)
+		require.NoError(t, err)
+		spec, ok := updated.Object["spec"].(map[string]any)
+		require.True(t, ok)
+		title, ok := spec["title"].(string)
+		require.True(t, ok)
+		require.Equal(t, first.GetName(), updated.GetName())
+		require.Equal(t, first.GetUID(), updated.GetUID())
+		require.Equal(t, "Test folder (replaced from k8s; 1 item; PUT)", title)
```

Comment:
Could we test the description field too?

Reply:
Good idea. Added it here: [4022e4f](https://github.com/grafana/grafana/pull/95033/commits/4022e4fd844f76d99df2cb9aa2c8ebf436e1ce37)

Reply:
Thank you!